### ISSUE TITLE
Tests fail (without XSConfig?)

### ISSUE THREAD

**Issue by eserte** [perl11/Mock-Config#1]
See subject. A fail looks like this:

```
PERL_DL_NONLAZY=1 /opt/perl-5.12.1/bin/perl "-MExtUtils::Command::MM" "-e" "test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
Use of uninitialized value $Config::VERSION in concatenation (.) or string at t/0-basic.t line 15.
# Testing Mock::Config, Perl 5.012001, Config , /opt/perl-5.12.1/bin/perl
Can't use an undefined value as a HASH reference at /var/tmp/cpan-build/2016022815/Mock-Config-0.01-ZUYVjb/blib/lib/Mock/Config.pm line 86.
# Looks like you planned 3 tests but ran 2.
# Looks like your test exited with 255 just after 2.
t/0-basic.t ....... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/3 subtests 
``
```

**Comment by rurban** [comment]
Ah thanks. I haven't tested it properly yet. Will do.

**Comment by rurban** [comment]
Released the fix as 0.02
